assisted-events-scrape/repositories/event_repository.py

```diff
@@ -0,0 +1,22 @@
+import json
```

Comment:
We have moved here all code related to retrieving events.
This should be changed in the future, but for now is as it was before.